web/resources/views/next/wiki/page.blade.php

```diff
@@ -61,11 +65,6 @@
             </div>
         @endif
 
-        {{-- TODO: proper 404 page --}}
-        {{--
-            Wikidot handles missing pages basically by showing hardcoded content
-            in the $page_content variable. That is stupid.
```

Comment:
lmao it did?

Reply:
yes. thank god it's dead. we will have a proper 404 page.